service/header/store.go

```diff
@@ -17,14 +17,39 @@ var (
 	DefaultStoreCacheSize = 1024
 	// DefaultIndexCacheSize defines the amount of max entries allowed in the Height to Hash index cache.
 	DefaultIndexCacheSize = 256
+	// DefaultWriteBatchSize defines the size of the batched header write.
+	// Headers are written in batches not to thrash the underlying Datastore with writes.
+	DefaultWriteBatchSize = 2056
```

Comment:
2056 headers? 2056 bytes? Also why 2056???

Reply:
Headers.

Reply:
Hmm, I have no idea why I put 2056 😅. I wanted it to be 2048, lol

Reply:
Made it 2048.